config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```diff
@@ -61,20 +62,28 @@ periodics:
         org: ppc64le-cloud
         repo: kubetest2-plugins
         workdir: true
+      - base_ref: master
+        org: kubernetes
+        repo: kubernetes
     spec:
       containers:
         - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20241021-d3a4913879-1.31
+          securityContext:
+            privileged: true
           env:
             - name: "BOSKOS_HOST"
               value: "boskos.test-pods.svc.cluster.local"
           resources:
             requests:
               cpu: "1500m"
+              memory: "5Gi"
```

Comment:
any reason for setting the memory limits here? job will not run if they don't have this much of memory, as the memory need will be high only during the initial build process and later it won't need lot of memory so I thinking removing memory limit will be better I feel.

Reply:
@mkumatag I faced pod eviction due to lack of memory during test runs. Hence had to make sure this pod has guaranteed memory.
I see arm jobs using 10Gi for their conformance jobs. https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/aws/ec2-e2e.yaml

Reply:
my only concern is to have those k8s node with this capacity, we may need to resize the nodes to higher levels if we run short of..

Reply:
Each node seems to have 16G size.
I thought 5G for now in one job wouldn't be a problem.